sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryListenersConfSuite.scala

```diff
@@ -31,7 +32,10 @@ class StreamingQueryListenersConfSuite extends StreamTest with BeforeAndAfter {
 
   override protected def sparkConf: SparkConf =
     super.sparkConf.set(STREAMING_QUERY_LISTENERS.key,
-      "org.apache.spark.sql.streaming.TestListener")
+      "org.apache.spark.sql.streaming.TestListener," +
```

Comment:
Let's fix this one together since we're here as `classOf[TestListener].getCanonicalName`.

Maybe sth like:

```scala
Seq(
  classOf[TestListener].getCanonicalName,
  classOf[TestSQLConfStreamingQueryListener].getCanonicalName).mkString(",")
```